--- build_sdk.py	2021-12-25 12:53:27.000000000 +0900
+++ build_sdk_fixed.py	2022-01-26 04:05:25.000000000 +0900
@@ -138,7 +138,7 @@
     def __str__(self):
         return "%s (%s)" % (self.name, self.toolchain)
     def haveIPP(self):
-        return self.name == "x86" or self.name == "x86_64"
+        return False
 
 #===================================================================================================
 
